tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -56,6 +62,40 @@ const DAY_IN_MS = 24 * 1000 * 60 * 60;
         </g>
       </svg>
     </div>
+    <mat-menu
+      #manualControl="matMenu"
+      xPosition="before"
+      [yPosition]="axis === 'y' ? 'above' : 'below'"
+    >
+      <div
+        class="extent-edit-input"
+        (click)="$event.stopPropagation()"
+        (keydown)="keydownPreventClose($event)"
+      >
+        <label>min</label>
+        <input #minInput tabindex="1" type="number" [value]="axisExtent[0]" />
```

Comment:
Panning the graph via alt+drag seems to set the value to extremely high precision, and by default the beginning of the number is not visible.  Can we round values when the form is hydrated?

![image](https://user-images.githubusercontent.com/2322480/109553995-8aef1f80-7a88-11eb-9061-88244ec3cfce.png)

Reply:
We can optimize those UX consideration in a follow up. Depending on a value, rounding can be quite janky thus is not a simple one or two line change.